    @Test
    public void testCreateIllegal() throws IOException {
        // for some combinations we expected exceptions
        try {
            assertNotNull(new CellRangeAddress(1, 0, 0, 0));
            fail("Expect to catch an exception");
        } catch (IllegalArgumentException e) {
            // expected here
        }
        try {
            assertNotNull(new CellRangeAddress(0, 0, 1, 0));
            fail("Expect to catch an exception");
        } catch (IllegalArgumentException e) {
            // expected here
        }
    }

